src/main/java/ch/njol/skript/conditions/CondStartsEndsWith.java

```diff
@@ -34,15 +34,15 @@
 
 @Name("Starts/Ends With")
 @Description("Checks if a text starts or ends with another.")
-@Examples({"if the argument starts with \"test\":",
-		"	send \"Stop!\""})
-@Since("2.2-dev36")
+@Examples({"if the argument starts with \"test\" or \"debug\":",
+	"	send \"Stop!\""})
```

Comment:
Can you replace the tab here with `\t`? Since you're already editing this expression ;D